docs/microsoft_entra_workload_id.md

```diff
@@ -0,0 +1,555 @@
+```
+
+Verify the ClusterImageSet:
+```bash
+$ oc get cluster-imageSet
+NAME                   RELEASE
+cluster-imageset   quay.io/openshift-release-dev/ocp-release:4.19.12-x86_64
+```
+
+#### 6.4 Create Azure Cloud Credentials Secret
+
+Refer to: [Using Hive - Cloud Credentials](using-hive.md#cloud-credentials)
+
+Create the Azure credentials secret:
+
+```bash
+$ oc create secret generic azure_creds -n <namespace> --from-file=/<azure_credentials_file_path>/osServicePrincipal.json 
+```
+
+#### 6.5 Create InstallConfig Secret
+
+Refer to: [Using Hive - InstallConfig](using-hive.md#installconfig)
+
+Prepare install-config.yaml using openshift-install
+
+```bash
+Example:
+$ ./openshift-install create install-config --dir <install-config-dir>
+? SSH Public Key /home/cloud-user/.ssh/id_rsa.pub
+? Platform azure
+INFO Credentials loaded from file "/<azure_credentials_file_path>/osServicePrincipal.json" 
+? Region <azure_region>
+? Base Domain <base_domain>
+? Cluster Name <cluster_name>
+? Pull Secret [? for help] ********************************************************************************************************************************************
+INFO Install-Config created in: <install-config-dir> 
+
+$ ls <install-config-dir>
+Install-config.yaml
+```
+
+Set credentialsMode to Manual:
+
+```bash
+$ echo "credentialsMode: Manual" >> <install-config-dir>/install-config.yaml
+```
+
+Set platform.azure.resourceGroupName to the installation resource group created in Step 2:
+
+```bash
+$ sed -i '/region: centralus/a\    resourceGroupName: <azure_infra_name>' <install-config-dir>/install-config.yaml
+```
+
+Create install-config secret:
+
+```bash
+$ oc create secret generic install-config-secret --from-file=install-config.yaml=<install-config-dir>/install-config.yaml
+```
+
+#### 6.6 Deploy ClusterDeployment
+
+Refer to: [Using Hive - ClusterDeployment](using-hive.md#clusterdeployment)
+
+**Important configurations:**
+- Set `spec.boundServiceAccountSigningKeySecretRef.name` to point to the Secret created in Step 4 (`bound-service-account-signing-key`)
+- Set `spec.provisioning.manifestsSecretRef` to point to the Secret created in Step 5 (`cluster-manifests`)
+- Set `spec.provisioning.imageSetRef` to point to the ClusterImageSet created in Step 6.3 (`cluster-imageset`)
+- Set `platform.azure.credentialsSecretRef.name` to point to the Secret created in Step 6.4 (`azure_creds`)
+- Set `spec.provisioning.installConfigSecretRef` to point to the Secret created in Step 6.5 (`install-config-secret`)
+
+**Example ClusterDeployment YAML:**
+
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  creationTimestamp: null
+  name: cluster-name
+spec:
+  baseDomain: base_domain
+  boundServiceAccountSigningKeySecretRef:
+    name: bound-service-account-signing-key
+  clusterName: cluster-name
+  controlPlaneConfig:
+    servingCertificates: {}
+  installed: false
+  platform:
+    azure:
+      baseDomainResourceGroupName: xxx
```

Comment:
I don't know if this is important to mention, but I used the baseDomainResourceGroupName mentioned in my generated install config